schedule/yast/raid/raid1_opensuse_lvm.yaml

```diff
@@ -0,0 +1,35 @@
+---
```

Comment:
the name of the test is `raid1_lvm`, so I would name it: `raid1_lvm_opensuse.yaml` instead

Reply:
no problem but i was trying to follow the name convention from other files in the directory. for instance another one is raid1_opensuse_gpt.yaml

Reply:
you're right, leave like that then...